src/worker.py

```
@@ -1329,6 +1331,321 @@ async def serve_static(path: str, env):
+                "display_name": info["display_name"],
+                "text": text,
+                "timestamp": data.get("timestamp", ""),
+            }))
+
+        elif msg_type == "update_seat":
+            seat_id = data.get("seat_id", "")
+            if not seat_id:
+                return
+
+            # Check if seat is occupied by someone else
+            for other_sid, other_info in self.sessions.items():
```

Ruff (0.15.10)

[warning] Loop control variable `other_sid` not used within loop body

Rename unused `other_sid` to `_other_sid`

(B007)